.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 .DS_Store
+.idea
```

Comment:
This should be in your global.gitignore and not the project

Reply:
Done.